tests/python/gaia-ui-tests/gaiatest/tests/functional/messages/manifest.ini

```diff
@@ -74,3 +74,6 @@ stable = false
 skip-if = device == "desktop"
 dogfood = true
 external = true
+
+[test_sms_report.py]
+skip-if = device == "desktop"
```

Comment:
I see you added tests/python/gaia-ui-tests/gaiatest/tests/functional/messages/__init__.py
and tests/python/gaia-ui-tests/gaiatest/tests/functional/__init__.py
.
Those are not needed, you should remove those files.